static/js/flow/emailTypes.js

```diff
@@ -31,5 +29,5 @@ export type EmailConfig = {
   title: string,
   renderSubheading: (subheading: string) => React$Element<*>,
   emailOpenParams: (args: any) => Object,
-  emailSendAction: (emailState: EmailState) => Dispatcher<EmailSendResponse>
+  emailSendParams: (emailState: EmailState) => Array<any>
```

Comment:
it can literally be anything. for one implementation, it's a complex ES query object. for another implementation, it's an integer ID